plugin-server/src/ingestion/ingestion-consumer.ts

```diff
@@ -114,10 +114,8 @@ export class IngestionConsumer {
     }
 
     public async start(): Promise<void> {
-        // NOTE: This needs to be started before the kafka consumer starts as other things rely on it
-        await this.hogTransformer.start()
-
         await Promise.all([
+            this.hogTransformer.start(),
```

Comment:
NOTE: This no longer needs to be done first as it is only starting the pubsub part for updates, whereas before it was loading all fns into memory